src/conn/conn_api.c

```diff
@@ -1115,6 +1115,11 @@ __conn_close(WT_CONNECTION *wt_conn, const char *config)
     conn->cache->eviction_dirty_trigger = 1.0;
     conn->cache->eviction_dirty_target = 0.1;
 
+    F_CLR(conn, WT_CONN_MINIMAL | WT_CONN_READY);
```

Comment:
Should we clear both flags before the `WT_EVENT_CONN_CLOSE` handler, or do we want to allow the same set of operations as in the `WT_EVENT_CONN_READY` handler? I.e., should we set `WT_CONN_MINIMAL` here?

Or maybe there is a different set of things that are/aren't allowed and we use the fact that neither `WT_CONN_MINIMAL` nor `WT_CONN_READY` are set to know that?

Reply:
Fixed. Clearing after the event handler now.